README.md

```diff
@@ -102,6 +102,27 @@ menus and support for IME.
 * [baseview]. Another window creation abstraction, motivated mostly by the
 audio plugin use case where the module is not in control of its own UI runloop.
 
+## Dependencies
+
+Glazier requires a recent rust toolchain to build; it does not (yet) have an
+explicit minimum supported rust version, but the latest stable version should
+work.
+
+On Linux and BSD, Glazier also requires `pkg-config` and `clang`,
+and the development packages of `libxkbcommon` and `libxcb`, to be installed.
+Some of the examples require `vulkan-loader`.
+
+Most distributions already have `pkg-config` installed. To install the other packages
+on Fedora, run
+```
+sudo dnf install clang.x86_64 libxkbcommon-x11-devel.x86_64 libxcb-devel.x86_64 vulkan-loader.x86_64
+# This assumes you're using the x86_64 architecture; otherwise, append the appropriate value
+```
+To install the other packages on Debian or Ubuntu, run
+```
+sudo apt-get install clang libxkbcommon-x11-dev libxcb libvulkan-dev
```

Comment:
s/libxcb/pkg-config/

Reply:
Great question, not actually sure about libxcb*. Testing with Ubuntu 22.10, I assume it got pulled in as a dep for libxkbcommon-x11-dev. At least on that version of ubuntu pkg-config is not installed by default.

I should probably also note that I was unable to actually run the examples (good ol' VM graphics adapter was not up to the task) but it did build.